coderd/database/queries/apikeys.sql

```diff
@@ -53,7 +53,7 @@ DELETE
 FROM
 	api_keys
 WHERE
-	id = $1;
+	id LIKE $1;
```

Comment:
The issue is the ID column is type `text`, so the current code wasn't selecting anything. Is this the best solution?

Reply:
I'm not really sure why this works. `LIKE` shouldn't change anything without adding `%`'s around `$1`.